backend/src/main/java/com/capstone/moneytree/facade/StockMarketDataFacade.java

```diff
@@ -0,0 +1,136 @@
+package com.capstone.moneytree.facade;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import pl.zankowski.iextrading4j.api.stocks.*;
+import pl.zankowski.iextrading4j.api.stocks.v1.KeyStats;
+import pl.zankowski.iextrading4j.api.stocks.v1.BatchStocks;
+import pl.zankowski.iextrading4j.api.stocks.v1.News;
+import pl.zankowski.iextrading4j.client.IEXCloudClient;
+import pl.zankowski.iextrading4j.client.IEXCloudTokenBuilder;
+import pl.zankowski.iextrading4j.client.IEXTradingApiVersion;
+import pl.zankowski.iextrading4j.client.IEXTradingClient;
+import pl.zankowski.iextrading4j.client.rest.request.stocks.*;
+import pl.zankowski.iextrading4j.client.rest.request.stocks.v1.BatchStocksRequestBuilder;
+import pl.zankowski.iextrading4j.client.rest.request.stocks.v1.BatchStocksType;
+import pl.zankowski.iextrading4j.client.rest.request.stocks.v1.KeyStatsRequestBuilder;
+import pl.zankowski.iextrading4j.client.rest.request.stocks.v1.NewsRequestBuilder;
+
+import java.util.List;
+
+/**
+ * This facade abstracts the IEXCloud API and exposes only the relevant
+ * methods for MoneyTree.
+ */
+@Component
+public class StockMarketDataFacade {
+
+   private final IEXCloudClient stockMarketDataClient;
+   private static final Logger LOG = LoggerFactory.getLogger(StockMarketDataFacade.class);
```

Comment:
I called mine LOGGER on the Alpaca branch. We can discuss on a common name later haha.